docs/src/less/pygments.less

```diff
@@ -0,0 +1,66 @@
+.highlight .hll { background-color: #ffffcc }
+
+.highlight .c { color: #999; }
+.highlight .err { color: #aa0000; background-color: #ffaaaa }
+.highlight .k { color: #006699; }
+.highlight .o { color: #555555 }
```

Comment:
I see both forms of specifying the color hex code. `#999` and `#555555`. Should we pick one style?

Reply:
This CSS is generated from [Pygments](http://pygments.org). I don't do any post clean-up on it. I just drop it straight in.